backend/lib/pipeline.ml

```diff
@@ -95,7 +94,9 @@ let pipeline ~slack_path ~conninfo ?branch ?pull_number ~dockerfile ~tmpfs
     let () =
       let db = new Postgresql.connection ~conninfo () in
       output
-      |> Json_utils.parse_many
+      |> Yojson.Safe.from_string
```

Comment:
The `result` member parsing here looks good, but the ability to support multiple objects is removed.

Essentially what we want is to:

1. Parse "many" JSON objects.
2. Extract "name" from the root of the object as ~benchmark_name.
3. Iterate on "results" from each object and make benchmark values.

Reply:
Thanks, I adapted the parsing code :)